.github/workflows/pr-verify.yaml

```diff
@@ -5,15 +5,64 @@ on:
     types: [opened, edited, synchronize, reopened]
 
 permissions:
-  checks: write # Allow access to checks to write check runs.
+  contents: read
+  pull-requests: write
+
+env:
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
 jobs:
-  verify:
+  verify-pr-type:
```

Comment:
Let's combine the two jobs into just one (I assume two jobs lead to two "checks" on the PR, less checks is better here)